### PR TITLE
[Mellanox]Add NVIDIA Copyright header to "device\mellanox" files

### PR DESCRIPTION
**Pull request by dprital**
Signed-off-by: Dror Prital <drorp@nvidia.com>

#### Why I did it

Add NVIDIA Copyright header to "device\mellanox" files

#### How I did it

Add NVIDIA Copyright header as a comment for Mellanox device files 

#### How to verify it

Run regression tests 

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106

### CODE REVIEW

File: device/mellanox/x86_64-mlnx_msn4600c-r0/ACS-MSN4600C/sai_4600C.xml [hunk 1]
```diff
@@ -467,4 +483,4 @@
             </port-info>
         </ports-list>
     </platform_info>
-</root>
```
Comment (dgsudharsan) [inline device/mellanox/x86_64-mlnx_msn4600c-r0/ACS-MSN4600C/sai_4600C.xml:470]: Can we remove this extra change?

File: device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700-D48C8/sai_2700_48x50g_8x100g.xml [hunk 2]
```diff
@@ -1,270 +1,286 @@
-<?xml version="1.0"?>
```
Comment (dgsudharsan) [inline device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700-D48C8/sai_2700_48x50g_8x100g.xml:1]: Any reason why the entire file is shown as diff?

**Review by dgsudharsan — CHANGES_REQUESTED** [review]

File: device/mellanox/x86_64-mlnx_msn4600c-r0/ACS-MSN4600C/sai_4600C.xml (hunk 1 above)
Comment (dprital) [reply device/mellanox/x86_64-mlnx_msn4600c-r0/ACS-MSN4600C/sai_4600C.xml:470]: I see that the </root> was removed and added right after. I don't know why it was removed and re-added, the changes derived from a tool

File: device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700-D48C8/sai_2700_48x50g_8x100g.xml (hunk 2 above)
Comment (dprital) [reply device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700-D48C8/sai_2700_48x50g_8x100g.xml:1]: Good question. I don't know.

**Review by dgsudharsan — APPROVED** [review]

**Pull request closed without merging**